src/component/Person.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Person.scss';
+
+const Person = ({ name, isMarried, sex, partnerName, age }) => (
+  <section className="Person__name">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+
+    { age ? (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    ) : (
+      <span />
+    )
+    }
+
+    { isMarried === true ? (
+      <p className="Person__partner">
+        My
+        {` ${sex === 'm' ? 'wife' : 'husband'}'s `}
+        name is
+        {` ${partnerName}`}
+      </p>
+    )
+      : (
+        <p className="Person__partner">I am not married</p>
+      )
+      }
```

Comment:
Fix code style
It's better to create additional variables before return to store wife/husband value and use it in your jsx